jina/util/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""Module for Jina helper functions for batching documents in executors."""
```

Comment:
Delete this file.

In general, we do not prefer some "misc"/"util" module to store orphan methods, it is not part of our design. In Jina, every method should have a clear parent (at least to the best reasonable effort).